pkg/management/component.go

```diff
@@ -361,6 +403,15 @@ func (c *component) UpdateUser(ctx context.Context, realmName, userID string, us
 	}
 	keycloakb.ConvertLegacyAttribute(&oldUserKc)
 
+	// get the "old" user infos from database
+	var oldDbUser *dto.DBUser = &dto.DBUser{}
```

Comment:
You should set UserID in case user is not found in DB

Reply:
I am asking my self if it should not be the job of the module to return such stuct... Would you be against such modification ?

Reply:
If the user is not found, it would be strange to return a default DBUser value without information that user was not found.